project/Settings.scala

```diff
@@ -90,22 +91,33 @@ object Settings {
       addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.0")
     )
 
+  lazy val entrypoint = "/opt/docker/bin/entrypoint.sh"
+
   lazy val commonDockerSettings = List(
     maintainer := "workbench-interactive-analysis@broadinstitute.org",
     dockerBaseImage := "oracle/graalvm-ce:1.0.0-rc16",
     dockerRepository := Some("us.gcr.io"),
     dockerExposedPorts := List(8080),
     dockerUpdateLatest := true,
+    // See https://www.scala-sbt.org/sbt-native-packager/formats/docker.html#add-commands
     dockerCommands ++= List(
-      ExecCmd("CMD", "export", "CLASSPATH=lib/*jar")
```

Comment:
I removed this `CMD` because it seemed like the classpath was already set, and it doesn't seem like the correct usage of docker `CMD` anyway

Reply:
hmmm weird..I thought I ran into jar can't find without this.